### PR TITLE
added 'k' key execution

### LINKED ISSUE #113: hotfix: 'k' key not working properly
- added 'k' key behaviour, which was missing on `develop` branch.

### PR DESCRIPTION
**Pull request by Henrique1792**
Fixes #113

This PR proposes the following changes:
- added 'k' movement

### CODE REVIEW

**Review by Float07 — CHANGES_REQUESTED** [review]
(this comment was a mistake. Ignore this 😅. The comment bellow with the suggestion is the right one)

File: src/ttsnake.c [hunk 1]
```diff
@@ -498,6 +498,7 @@ void * userinput(){
 				break;
       			case KEY_UP:
       			case 'w':
+				case 'k':
 			case 'e':
```
Comment (Float07) [inline src/ttsnake.c]: We have one extra tab here:
```suggestion
      			case 'w':
			case 'k':
			case 'e':
```

**Review by Float07 — APPROVED** [review]
Just a small observation

**Comment by CarlosCraveiro** [comment]
Great pull request!! I had implemented the "h j k l" feature, but someone accidentally removed the "k" key during some other pull request.

**Review by CarlosCraveiro — APPROVED** [review]

**Review by Dauboau — APPROVED** [review]

**Pull request merged**